bears/vcs/git/GitCommitBear.py

```diff
@@ -241,7 +241,7 @@ def check_body(self, body,
                                'HEAD commit. Please add one.')
             return
 
-        if body_regex and not re.search(body_regex, body):
+        if body_regex and not re.fullmatch(body_regex, body.strip()):
```

Comment:
@jayvdb I think `strip` is alright, git should guarantee that only a traling \n is there. Also when someone writes commit messages with 10 newlines between body and shortlog, we catch it nicely away. Imo `body.strip()` contains then the real commit body :+1: